From 930eae93f066fc78a5e96c88ca91536748f891d3 Mon Sep 17 00:00:00 2001
From: Peter Chen <peter.chen@nxp.com>
Date: Sun, 22 Jan 2017 10:50:09 +0800
Subject: [PATCH 1485/5242] MLK-13760 ARM: dts: imx7ulp-evk: update tx-d-cal
 according to certification results

commit  306d1a66173e7e5ec3b2cd82b1584cadcdb403a4 from
https://source.codeaurora.org/external/imx/linux-imx.git

Changing tx-d-cal according to USB certification test results.

Signed-off-by: Peter Chen <peter.chen@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx7ulp-evk.dts |    4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm/boot/dts/imx7ulp-evk.dts b/arch/arm/boot/dts/imx7ulp-evk.dts
index dedd859..9d2d2c5 100644
--- a/arch/arm/boot/dts/imx7ulp-evk.dts
+++ b/arch/arm/boot/dts/imx7ulp-evk.dts
@@ -435,6 +435,10 @@
 	status = "okay";
 };
 
+&usbphy1 {
+	tx-d-cal = <0xc>;
+};
+
 &usdhc0 {
 	pinctrl-names = "default", "state_100mhz", "state_200mhz", "sleep";
 	pinctrl-0 = <&pinctrl_usdhc0>;
-- 
1.7.9.5

